addons/mass_mailing/i18n/mass_mailing.pot

```diff
@@ -4,10 +4,10 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 16.0beta\n"
+"Project-Id-Version: Odoo Server 16.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-10-10 10:22+0000\n"
-"PO-Revision-Date: 2022-10-10 10:22+0000\n"
+"POT-Creation-Date: 2022-10-22 16:18+0000\n"
```

Comment:
We don't change anything else than added strings to translate :) could you revert those changes and keep only the new string ?